libenkf/src/queue_config.c

```diff
@@ -57,6 +58,37 @@ queue_config_type * queue_config_alloc() {
     return queue_config;
 }
 
+queue_config_type * queue_config_alloc_local_copy( queue_config_type * queue_config) {
+    queue_config_type * queue_config_copy = util_malloc(sizeof * queue_config_copy);
+    queue_config_copy->queue_drivers = hash_alloc();
+    queue_config_add_queue_driver(queue_config_copy, LOCAL_DRIVER_NAME, queue_driver_alloc_local());
+    queue_config->user_mode = false;
+
+    if (queue_config_has_job_script(queue_config)) {
+        queue_config_copy->job_script = util_alloc_copy(queue_config->job_script, strlen( queue_config->job_script) + 1);        
+    }
+    else
+        queue_config_copy->job_script = NULL;
+
+    if (queue_config->user_mode) {
+        queue_config_init_user_mode( queue_config_copy );
+    }
+
+    if (queue_config->driver_type == NULL_DRIVER)
```

Comment:
No need to check for `NULL_DRIVER` - just set to `LOCAL_DRIVER` unconditionally.